docs/recipes/package-management/migrate-to-paket.md

```diff
@@ -5,19 +5,18 @@ It can help you reference libraries from NuGet, Git repositories or Http resourc
 
 >  Note that the standard template uses Paket by default. **This recipe only applies to the minimal template**.
 
-
 ---
 
-#### 1. Create a .NET Tool Manifest
-The manifest allows others to easily install the tools required to build your application. At the root of your codebase run:
+#### 1. Install and restore Paket
 ```bash
-dotnet new tool-manifest
+dotnet tool install paket
+dotnet tool restore
 ```
 
-#### 2. Install and restore Paket
+#### 2. Run the Migration
```

Comment:
Is this mandatory?

Reply:
It is if you want to follow the steps to get Fake and Farmer working, but it looks like they have been removed.